pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1724,7 +1726,10 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 						Issue in Libvirt: https://gitlab.com/libvirt/libvirt/-/issues/608
 						once the issue is resolved we can remove mpx disablement
 		*/
-		if _, exists := existingFeatures["mpx"]; !exists && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostModel && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostPassthrough {
+
+		_, exists := existingFeatures["mpx"]
+		// skip the mpx CPU feature validation for s390x as it is not supported.
+		if !checks.IsS390X(runtime.GOARCH) && !exists && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostModel && vmi.Spec.Domain.CPU.Model != v1.CPUModeHostPassthrough {
```

Comment:
I wonder if this check would affect unit test somehow

Reply:
Actually, all the unit test jobs are passing. I don't think this check affected any unit tests.

Reply:
Did you run them with a s390x as well?

Reply:
yes

Reply:
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/13050/pull-kubevirt-unit-test-s390x/1844595619528708096
This job will test the unit tests on s390x.